src/App.js

```diff
@@ -13,13 +13,11 @@ import CardDetail from './components/CardDetail/CardDetail';
 function App() {
   const dispatch = useDispatch();
 
-  const fetchCards = useCallback(async () => {
-    const response = await axios(
+  const fetchCards = () => {
+    axios(
       'https://raw.githubusercontent.com/BrunnerLivio/PokemonDataGraber/master/output.json'
-    );
-
-    dispatch(
-      cardListActions.setCards(
+    )
+      .then((response) =>
         response.data.slice(0, 15).map((cardData) => {
           return {
             id: v4(),
```

Comment:
Could you please move external call into a separate module? (api.js for example)